packages/node/src/generators/application/application.ts

```diff
@@ -211,7 +211,7 @@ export async function applicationGenerator(tree: Tree, schema: Schema) {
       setupFile: 'none',
       skipSerializers: true,
       supportTsx: options.js,
-      babelJest: options.babelJest,
+      transformer: options.babelJest ? 'babel-jest' : 'ts-jest',
```

Comment:
suggestion: we should update the schema of all the projects that have `--babelJest` as an option to have the `transformer` enum, so those consuming the jest project generator can utilize swc/jest if the option is passed through.